public/app/features/alerting/unified/features.ts

```diff
@@ -6,6 +6,7 @@ export enum AlertingFeature {
   NotificationPoliciesV2MatchingInstances = 'notification-policies.v2.matching-instances',
   DetailsViewV2 = 'details-view.v2',
   ContactPointsV2 = 'contact-points.v2',
+  InsightsPage = 'insights-page',
```

Comment:
@VikaCep in the future please use the built-in feature toggle system Grafana has, as far as I am aware all other teams use it. With it you can create feature toggles that can be set in the ini file, via ENV variables, and via URL query parameter, and viewed in the UI. 

https://github.com/grafana/grafana/blob/main/pkg/services/featuremgmt/registry.go

Reply:
Good call. Changed it to use the built-in feature toggle in https://github.com/grafana/grafana/pull/74749